packages/react-instantsearch/src/core/createInstantSearchManager.error.test.js

```diff
@@ -51,21 +48,26 @@ describe('createInstantSearchManager', () => {
 
         expect(ism.store.getState().error).toBe(null);
 
-        jest.runAllTimers();
+        return Promise.resolve().then(() => {
+          jest.runAllTimers();
```

Comment:
Yes the runAllTimers is for the JS client mock (setImmediate), the return Promise is for the widget manager. Works?